README.md

```diff
@@ -72,10 +72,37 @@ can be found [here](lib)
 
 	
 ## Documentation
+
 Detailed documentation can be found [here](docs).
-Usage examples can be found [here](samples)
-	
-## Contributing 
 
-Contributions and suggestions to this product will be possible, and most welcome, 
-starting with next release
\ No newline at end of file
+## Samples
+
+Usage examples can be found [here](samples/src/main/scala)
+
+# Looking for SDKs for other languages/platforms?
+
+- [Node](https://github.com/azure/azure-kusto-node)
+- [Python](https://github.com/azure/azure-kusto-python)
+- [.NET](https://docs.microsoft.com/en-us/azure/kusto/api/netfx/about-the-sdk)
+- [Java](https://github.com/azure/azure-kusto-java)
+
+# Need Support?
+
+- **Have a feature request for SDKs?** Please post it on [User Voice](https://feedback.azure.com/forums/915733-azure-data-explorer) to help us prioritize
+- **Have a technical question?** Ask on [Stack Overflow with tag "azure-data-explorer"](https://stackoverflow.com/questions/tagged/azure-data-explorer)
+- **Need Support?** Every customer with an active Azure subscription has access to [support](https://docs.microsoft.com/en-us/azure/azure-supportability/how-to-create-azure-support-request) with guaranteed response time.  Consider submitting a ticket and get assistance from Microsoft support team
+- **Found a bug?** Please help us fix it by thoroughly documenting it and [filing an issue](https://github.com/Azure/azure-kusto-spark/issues/new).
+
+# Contributing
```

Comment:
Thanks for adding this.
Can we add a document with more specific guidelines for contributors, and link from here? Is there anything like it in Java client?

Reply:
I didn't find general guidelines for contribution. This comment is exactly as the one of Java SDK.